master/internal/command/events.go

```diff
@@ -44,17 +48,70 @@ type event struct {
 	LogEvent *string `json:"log_event"`
 }
 
+type logSubscribers = map[*actor.Ref]webAPI.LogsRequest // CHECK could we get different pointers to
+// the same ref? thus resulting in different map keys
+
+// GetEventCount is an actor message used to get the number of events in buffer.
+type GetEventCount struct{}
+
 type eventManager struct {
-	bufferSize int
-	buffer     *ring.Ring
-	closed     bool
-	seq        int
+	bufferSize   int
+	buffer       *ring.Ring
+	closed       bool
+	seq          int
+	isTerminated bool // DISCUSS If we don't want to don't keep track of this here we'd either need to
+	// expose event externally or define and send a new message. also the logActor might not receive
+	// the termination log message (based on LogRequest) if we rely on exposing Event..
```

Comment:
Why would the logActor not receive the termination log message?

If it is paging through some middle part of the log, that's fine, isTerminated wouldn't matter.

If it is in a follow, then the log actor would either receive the terminated event, or a CloseStream message as the eventlog hit PostStop, right?

Reply:
> If it is paging through some middle part of the log, that's fine, isTerminated wouldn't matter.

right I think I got confused on this part